monstereo/visuals/pifpaf_show.py

```diff
@@ -72,12 +72,13 @@ def load_image(path, scale=1.0):
 class KeypointPainter(object):
     def __init__(self, *,
                  skeleton=None,
-                 xy_scale=1.0, highlight=None, highlight_invisible=False,
+                 xy_scale=1.0, y_scale=1.0, highlight=None, highlight_invisible=False,
```

Comment:
What do you use that for? Is it in the new version?

Reply:
When using multi mode, the bird view would move the image up and the keypoints would be displayed below the person. Therefore the y_scale of the keypoints had to be adjusted by the y_scale of the printer. I added this argument because in this case the x_scale should not change and also I didn't want to alter the previous behavior of the KeypointsPainter.